ui/src/components/TopBar.vue

```diff
@@ -53,7 +54,23 @@ export default {
       swagger_link: "/swagger/index.html"
     };
   },
+  computed: {
+    isAuthenticated: function() {
+      return this.$store.getters['auth/authenticated'];
+    }
+  },
+  watch: {
+    isAuthenticated() {
+      this.$router.push('/login');
+    }
+  },
   methods: {
+    logout() {
+      window.gapi.auth2.getAuthInstance().signOut().then(user => {
+        this.$store.dispatch('auth/updateUser', user);
+        sessionStorage.clear();
+      });
+    },
```

Comment:
I haven't tested this locally, but I assume this will also redirect the user back to the login page?